content/blog/2019-12-01-skeleton/index.Rmd

```diff
@@ -0,0 +1,95 @@
+---
+slug: post-template
+title: Wonderful title
+package_version: 0.1.0
+authors:
+  - Author Name
+date: 2019-06-04
+categories: blog
+topicid:
+tags:
+- Software Peer Review
+- R
+- community
+# delete the line below
```

Comment:
revise this comment to include what to do

"include link to preferred image for Twitter cards or delete the twitterImg line below if you ..."